lib/ansible/modules/blockinfile.py

```diff
@@ -287,12 +297,16 @@ def main():
     else:
         insertre = None
 
+    wrap_empty_line = b("")
+    if module.boolean(params['wrap_with_blank_lines']):
```

Comment:
The type has been validated at this point, so this could just be:
```suggestion
    if params['wrap_with_blank_lines']:
```

Reply:
True, I will fix it asap :smile: